src/projection/node/create-projection-node.ts

```diff
@@ -807,7 +814,11 @@ export function createProjectionNode<I>({
              * a relativeParent. This will allow a component to perform scale correction
              * even if no animation has started.
              */
-            if (!this.targetDelta && !this.relativeTarget) {
+            if (
+                this.safeToResolveRelativeTarget &&
```

Comment:
Why do we need this check? After the first time it should get a relative target right? 🤔

Reply:
Ah yeah I've removed it, I think this is not needed any more